craco.config.js

```diff
@@ -20,12 +39,15 @@ module.exports = {
         ],
       },
     },
-  },
-  devServer: {
-    watchOptions: {
-      ignored: '**/node_modules/**',
+    plugins: {
+      add: [
+        new webpack.ProvidePlugin({
+          Buffer: ['buffer', 'Buffer'],
+        }),
+      ],
     },
   },
+  devServer: {},
```

Comment:
Why remove this? Is the option no longer available?

Reply:
The option is different now and from what I can tell in the guts of create-react-app, it ignores node_modules by default.

Reply:
Just curious where you saw that it was different now. This page seems to describe the option exactly as it was set: https://webpack.js.org/configuration/watch/#watchoptionsignored

Reply:
It throws an error about an invalid configuration if you try to set it. It looks like it moved to be a general webpack option while the above setting is for the `devServer` specifically. There is `devServer` watchFiles but no ignore.

`craco` can be confusing - not sure where you would set `watch` options, under the `configure` object maybe?